internal/datanode/data_node.go

```diff
@@ -85,7 +85,7 @@ type DataNode struct {
 	cancel           context.CancelFunc
 	Role             string
 	stateCode        atomic.Value // commonpb.StateCode_Initializing
-	flowgraphManager *flowgraphManager
+	flowgraphManager *fgManagerImpl
```

Comment:
why not use `FlowgraphManager` interface here?

Reply:
Ture, forgot to change it.